applets/mount/mount_linux.go

```diff
@@ -20,14 +21,32 @@ func Mount(call []string) error {
 		return e
 	}
 
-	if flagSet.NArg() != 2 || *helpFlag {
-		println("`mount` [options] <device> <dir>")
-		flagSet.PrintDefaults()
-		println("\nAvailable options are:")
-		for opt := range flagMap {
-			print(opt, ", ")
+	if flagSet.NArg() < 1 && !*helpFlag {
+
+		// try mtab
+		mountinfo, err := ioutil.ReadFile("/etc/mtab")
```

Comment:
This file is usually distribution specific and normally a symlink to `/proc/mounts` anyway

Why not just check `/proc/mounts` and be done with it?

Reply:
in case there is no `procfs`

Reply:
If you have no procfs; then how will the presence of an `/etc/mtab` help? Normally on most systems you have this:

```#!sh
/ # ls -l /etc/mtab
lrwxrwxrwx    1 root     root            12 Nov 29 05:11 /etc/mtab -> /proc/mounts
```

Reply:
if there were no procfs, the `mount` command would have to keep `/etc/mtab` up to date anyways, which we aren't doing.

so yeah, it should probably simply print `/proc/mounts`

fwiw, the `mount` program i have installed reads `/proc/filesystems` and `/proc/self/mountinfo`